hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -180,6 +196,12 @@ public Response getClusterState() {
         .setTotalDatanodes(datanodeDetails.size())
         .setHealthyDatanodes(healthyDatanodes)
         .setOpenContainers(containerStateCounts.getOpenContainersCount())
+        .setClosingContainers(containerStateCounts.getClosingContainersCount())
+        .setQuasiClosedContainers(
+            containerStateCounts.getQuasiClosedContainersCount())
+        .setClosedContainers(containerStateCounts.getClosedContainersCount())
+        .setDeletingContainers(
+            containerStateCounts.getDeletingContainersCount())
         .setDeletedContainers(containerStateCounts.getDeletedContainersCount())
```

Comment:
This might be duplicate of PR for [HDDS-8127](https://github.com/apache/ozone/pull/4440)

Reply:
Deleted Container and Deleting container shows different info ... and not to be excluded from container count. Only Deleted Container needs to be excluded,